### ISSUE TITLE
Bug fixed version.

### ISSUE THREAD

**Issue by aa1234241** [dome272/VQGAN#25]
Hello, I have uploaded a [bug-fixed version](https://github.com/aa1234241/vqgan) of this implementation. 
Here is my result.
![283_1000](https://github.com/user-attachments/assets/d6397cf5-7004-4cea-9363-85b72a970f88)

The bugs that I've found:
Visualization error.
Perceptual loss error. The NetLinLayer does not have the right name, so the pre-trained model failed to load weights on them. I have fixed this issue by replacing it with the official VQGAN code.
Also, the decoder part in VQGAN model seems not correct, I replaced the encoder and decoder using the official VQGAN code.
For the gan loss. The disc-start should not be too early, the previous given value is 10000, replace it with 100000.
Also, the disc factor is too large, previous is 1, change it to 0.2
I also find the GAN loss becomes smaller as the training progresses. The lbd value converges to a very small value like 0.02 after 200 epochs.
The original codebook does not have a normalization operation. This causes many entries of the codebook to not be used during training. Add the embedding normalization made the codebook usage rage grow significantly.
